hongyujin/week10/4948.py

```diff
@@ -0,0 +1,23 @@
+def sosu(num):
+    for i in range(2, int(num ** 0.5) + 1):
+        if num % i == 0:
+            return False
+    return True
+
+
+sosu_list = []
+
+while True:
+    sosu_list.clear()
```

Comment:
```suggestion
    sosu_list = []
```
sosu_list 를 while문 내부에 선언하시면 미미한 변화지만 clear 함수를 안쓰셔도 될거에요..!!